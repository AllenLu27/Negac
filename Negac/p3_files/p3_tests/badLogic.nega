int fn(){
	int a;
	int b;
	a = a && b;
}
