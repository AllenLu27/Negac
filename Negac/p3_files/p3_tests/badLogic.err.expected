FATAL [4,6]: Logical operator applied to non-bool operand
FATAL [4,11]: Logical operator applied to non-bool operand
FATAL [4,4]: Invalid assignment operation
Semantic analysis failed
